chefs_soils.py

```diff
@@ -1,73 +1,1379 @@
+  "qualifiedProfessionalPhoneNumber",
+  "qualifiedProfessionalEmail",
+  "signaturerFirstAndLastName",
+  "dateSigned",
+  "createAt",
+  "confirmationId"
+]
+
+RECEIVING_SITE_HEADERS = [
+  "ownerFirstName",
+  "ownerLastName",
+  "ownerCompany",
+  "ownerAddress",
+  "ownerCity",
+  "ownerProvince",
+  "ownerCountry",
+  "ownerPostalCode",
+  "ownerPhoneNumber",
+  "ownerEmail",
+  "owner2FirstName",
+  "owner2LastName",
+  "owner2Company",
+  "owner2Address",
+  "owner2City",
+  "owner2Province",
+  "owner2Country",
+  "owner2PostalCode",  
+  "owner2PhoneNumber",
+  "owner2Email",
+  "additionalOwners",
+  "contactFirstName",
+  "contactLastName",
+  "contactCompany",
+  "contactAddress",
+  "contactCity",
+  "contactProvince",
+  "contactCountry",
+  "contactPostalCode",  
+  "contactPhoneNumber",
+  "contactEmail",
+  "SID",
+  "latitude",
+  "longitude",
+  "regionalDistrict",
+  "landOwnership",
+  "legallyTitledSiteAddress",
+  "legallyTitledSiteCity",
+  "legallyTitledSitePostalCode",
+  "PID",
+  "legalLandDescription",
+  "PIN",
+  "crownLandFileNumbers",  
+  "receivingSiteLandUse",
+  "CSRFactors",
+  "relocatedSoilUse",
+  "highVolumeSite",
+  "soilDepositIsALR",
+  "soilDepositIsReserveLands",
+  "dateSigned",
+  "createAt",
+  "confirmationId"
+]  
+
+HV_SITE_HEADERS = [
+  "ownerFirstName",
+  "ownerLastName",
+  "ownerCompany",
+  "ownerAddress",
+  "ownerCity",
+  "ownerProvince",
+  "ownerCountry",
+  "ownerPostalCode",
+  "ownerPhoneNumber",
+  "ownerEmail",
+  "owner2FirstName",
+  "owner2LastName",
+  "owner2Company",
+  "owner2Address",
+  "owner2City",
+  "owner2Province",
+  "owner2Country",
+  "owner2PostalCode",
+  "owner2PhoneNumber",
+  "owner2Email",
+  "additionalOwners",
+  "contactFirstName",
+  "contactLastName",
+  "contactCompany",
+  "contactAddress",
+  "contactCity",
+  "contactProvince",
+  "contactCountry",
+  "contactPostalCode",
+  "contactPhoneNumber",
+  "contactEmail",
+  "SID",
+  "latitude",
+  "longitude",
+  "regionalDistrict",  
+  "landOwnership",
+  "legallyTitledSiteAddress",
+  "legallyTitledSiteCity",
+  "legallyTitledSitePostalCode",
+  "PID",
+  "legalLandDescription",
+  "PIN",
+  "crownLandFileNumbers",
+  "receivingSiteLandUse",
+  "hvsConfirmation",
+  "dateSiteBecameHighVolume",
+  "howRelocatedSoilWillBeUsed",
+  "soilDepositIsALR",
+  "soilDepositIsReserveLands",
+  "qualifiedProfessionalFirstName",
+  "qualifiedProfessionalLastName",
+  "qualifiedProfessionalType",
+  "professionalLicenceRegistration",
+  "qualifiedProfessionalOrganization",
+  "qualifiedProfessionalAddress",
+  "qualifiedProfessionalCity",
+  "qualifiedProfessionalProvince",
+  "qualifiedProfessionalCountry",
+  "qualifiedProfessionalPostalCode",
+  "qualifiedProfessionalPhoneNumber",
+  "qualifiedProfessionalEmail",
+  "signaturerFirstAndLastName",
+  "dateSigned",
+  "createAt",
+  "confirmationId"
+]
+
+DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f%z'
+
+
+
+def convert_regional_district_to_name(id):
```

Comment:
could we please rename the `id` to `key` for all the functions
https://stackoverflow.com/questions/77552/id-is-a-bad-variable-name-in-python

Reply:
Renamed to key for all the functions.